scripts/entrypoint.sh

```diff
@@ -84,6 +89,9 @@ case ${OPTION} in
         ;;
     approve)
         cd /chi
+        mount -o bind /chi/config/backstop_data/bitmaps_reference/non_responsive /chi/reports/html_report/non_responsive_ce/bitmaps_reference
```

Comment:
@HectorGarciaAlvarez  This mount point should not be necessary because there is no approve task for Custom Elements (CE) version. Can we remove?

Reply:
👍